packages/mdc-snackbar/README.md

```diff
@@ -82,6 +82,16 @@ Action buttons with long text should be positioned _below_ the label instead of
 </div>
 ```
 
+Alternatively, you can call the `mdc-snackbar-layout-stacked` mixin from Sass:
+
+```scss
+@media (min-width: $mdc-snackbar-mobile-breakpoint) {
```

Comment:
max-width would probably be more appropriate for this example?

Reply:
Done